boostsec/registry_validator/upload_rules_db.py

```diff
@@ -112,17 +114,25 @@ def _get_namespace_and_driver(module: Path) -> tuple[str, str]:
     return namespace, driver
 
 
-def _get_rules_and_default(namespace: str, root: Path) -> tuple[RulesDB, Optional[str]]:
+def _get_rules_and_default(
+    namespace: str, config: RegistryConfig
+) -> tuple[RulesDB, Optional[str]]:
     """Get the rules and default rule if applicable."""
     if namespace == "default":
         namespace = "boostsecurityio/native-scanner"
-    rules_db_path = root / namespace / "rules.yaml"
-    rules_db_yaml = yaml.safe_load(rules_db_path.read_text())
+
+    scanners_path = config.scanners_path / namespace / "rules.yaml"
+    rules_realm_path = config.rules_realm_path / namespace / "rules.yaml"
+    if scanners_path.exists():
+        rules_db_yaml = yaml.safe_load(scanners_path.read_text())
+    else:
```

Comment:
if no scanner path exit and it falls be to checking in rules realm and it's still not there the yaml load would fail, but honestly should not happen unless there is a typo in the module 

you can fix it so that it doesn't throw an exception, but a clean error instead, but meh that's not the worse thing

Reply:
I am kind of leveraging that we already check that the import exists in the validation step, but I can always add it here too.